tests/functional/Ticket.php

```diff
@@ -1860,12 +1861,15 @@ public function testClone()
     {
         $this->login();
         $this->setEntity('Root entity', true);
-        $ticket = getItemByTypeName('Ticket', '_ticket01');
-
+        $ticket = new GlobalTicket();
```

Comment:
```suggestion
        // this test originally used the ticket created in the dataset 
        // this ticket contained a task, which on creation added the tech to the assigned actors 
        // and add  additional log entry relating to this action 
        // so $this->integer((int)$clonedTicket->getTimelineItems())->isEqualTo(0); return false 
        // because getTimelineItems also returns the history of the actors
        
        //$ticket = getItemByTypeName('Ticket', '_ticket01');
        
        $ticket = new GlobalTicket();
```